### PR TITLE
fix(icons): update `pentagon` icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?
- [x] Other: Icon update

### Description
Hand optimised corners to have 2px radius.

## Before Submitting
- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
I'm not super happy about the optical volume and centre of gravity of this icon, I feel like maybe it should be scaled up towards the top as such:
image

What do you think?

File: icons/pentagon.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../icon.schema.json",
   "contributors": [
-    "danielbayley"
+    "danielbayley",
+    "jguddas"
```
Comment (ericfennis) [inline icons/pentagon.json:5]: Name shouldn't be here, It's not a design change

Comment (jguddas) [reply icons/pentagon.json:5]: If you do manual changes/work that should count as a contribution IMO.

**Comment by ericfennis** [comment]
@karsa-mistmere To me it is not really centered if you place the icon in a single form.

**Review by ericfennis — DISMISSED** [review]

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere To me it is not really centered if you place the icon in a single form.

I find it to be a bit too bottom-heavy. 🤔

image

**Comment by ericfennis** [comment]
@karsa-mistmere ah, seeing this. I think I'm going with your proposal

**Review by ericfennis — APPROVED** [review]

**Pull request merged**